sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,29 +1,41 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2021-03-23)
 
 This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 
+### Breaking changes
+
 - The `getToken` methods will now never return `null`. If a token is not available, we will return a rejected promise.
-- `DefaultAzureCredential`'s implementation for browsers was simplified to throw a simple error instead of trying credentials that were already not supported for the browser.
-- Breaking Change: `InteractiveBrowserCredential` for the browser now requires the client ID to be provided.
-- Documentation was added to elaborate on how to configure an AAD application to support `InteractiveBrowserCredential`.
-- Replaced the use of the 'express' module with a Node-native http server, shrinking the resulting identity module considerably
-- Updated `@azure/msal-node-extensions` to [1.0.0-alpha.6](https://www.npmjs.com/package/@azure/msal-node-extensions/v/1.0.0-alpha.6).
+- `InteractiveBrowserCredential` for the browser now requires a client ID to be provided.
+- `InteractiveBrowserCredential` now only has `loginStyle` and `flow` in the optional parameters when the credential is bundled for browsers. This reflects the intended behavior, since the `loginStyle` has no effect on NodeJS.
+- Removed the `postLogoutRedirectUri` from the optional properties of the `InteractiveBrowserCredential`, to align with other languages.
```

Comment:
Do the other languages not have it because it is not required at all or just that they didnt have time to get to it yet?

Reply:
It's not required at all.

Reply:
Then, I would add that as the reason rather than alignment with other languages

Reply:
What about?

> Removed the `postLogoutRedirectUri` from the optional properties of the `InteractiveBrowserCredential`. This property is not required by our dependencies and it's not provided by the Identity SDKs in other languages.

Reply:
Looks like we used to pass this to the old msal. Does the new `msal-browser` not take this option anymore?

Reply:
It takes the option, but Scott requested me to remove this from the public API since none of the other languages took this in consideration. It's not needed for our use cases (but it's not useless. It is in fact useful)